api/build.gradle.kts

```diff
@@ -24,11 +24,13 @@ plugins {
 
 dependencies {
   implementation(libs.commons.lang3)
+  implementation(libs.commons.collections4)
   implementation(libs.guava)
   implementation(libs.slf4j.api)
 
   testImplementation(libs.junit.jupiter.api)
   testImplementation(libs.junit.jupiter.params)
+  testImplementation(libs.commons.collections4)
```

Comment:
I believe we can remove this line as it has been added to the run time in L27.

Reply:
fixed